stable/datadog/templates/daemonset.yaml

```diff
@@ -168,6 +168,14 @@ spec:
       {{- if .Values.daemonset.tolerations }}
       tolerations:
 {{ toYaml .Values.daemonset.tolerations | indent 8 }}
+      {{- end }}
+      {{- if .Values.daemonset.nodeSelector }}
+      nodeSelector:
+{{ toYaml .Values.daemonset.nodeSelector | indent 8 }}
```

Comment:
actually the nodeSelector part already exists https://github.com/snebel29/charts/blob/65e989c14377634d3b5c3dc6c0bbd05e39017b0b/stable/datadog/templates/daemonset.yaml#L181-L184